modules/apps/27-interchain-accounts/types/keys.go

```diff
@@ -9,10 +9,10 @@ const (
 	ModuleName = "interchainaccounts"
 
 	// PortID is the default port id that the interchain accounts host submodule binds to
-	PortID = "interchain-account"
```

Comment:
@damiannolan I think this will break compatibility with 0.9.0 hermes relayer, right? It's hardcoded to use `interchain-account`.

Reply:
I tested this out with hermes v0.9.0 and everything is working as expected. New controller port prefix in logs shows as below and account registers successfully:

```
2022-01-21T12:58:22.599864Z  INFO ThreadId(46) [test-1:icacontroller-cosmos1m9l358xunhhwds0568za49mzhvuxx9uxre5tud/channel-0 -> test-2] finished scheduling pending packets clearing height=Some(Height { revision: 1, height: 61 })
```

Reply:
On hermes `v0.10.0` you’ll see `modules/applications/mod.rs` which has a hardcoded prefix, but its not there on `v0.9.0`, neither is the custom version negotiation from `relayer/channel/version.rs`

Reply:
Wow. Thanks a lot, @damiannolan, for already testing this!